master/networking/dual-stack.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Enable dual stack
+canonical_url: 'https://docs.projectcalico.org/master/networking/dual-stack'
+---
+
+### Big picture
+
+Arrange for each Kubernetes pod to get both IPv4 and IPv6 addresses, so that it can communicate over
+both IPv4 and IPv6.
+
```

Comment:
Do we need to mention that Dual stack isn't GA in k8s?  And therefore that this Calico feature shouldn't be considered GA either?

Reply:
Done in next commit.